3ds2/src/main/java/com/adyen/checkout/adyen3ds2/internal/ui/model/Adyen3DS2ComponentParamsMapper.kt

```diff
@@ -62,6 +62,7 @@ internal class Adyen3DS2ComponentParamsMapper(
         if (sessionParams == null) return this
         return copy(
             amount = sessionParams.amount ?: amount,
+            shopperLocale = sessionParams.shopperLocale ?: shopperLocale,
```

Comment:
I would write a test for this mapper too.

Reply:
Good catch 👍 I added it